registry/templates/layout.html

```diff
@@ -4,7 +4,18 @@
   <head>
     <meta charset="utf-8" />
 
+    <!-- Favicon -->
     <link rel="shortcut icon" href="{{url_for('static', filename='favicon.ico') }}">
+
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
```

Comment:
Nejsem principiálně proti, ale raději bych tyto externí knihovny přidával postupně, jakmile je budeme potřebovat. Takže Bootstrap asi ano, ale jQuery a popper asi hned tak potřeba nebude. Nebo se pletu?

Reply:
Přiznávám, že tomu úplně nerozumím, prostě jsem postupovala podle návodu na stránkách Bootstrapu (https://getbootstrap.com/docs/4.5/getting-started/introduction/), který říká tohle dej tady a tohle tam a bude ti to fungovat.